core/modules/parser/QSMySqlListener.cc

```diff
@@ -797,6 +832,12 @@ class QuerySpecificationAdapter :
         _distinct = distinct;
     }
 
+    void checkContext() const override {
+        // required:
```

Comment:
What's a meaning of this 'required:'? I've seen it used in other similar methods.

Reply:
there's a description in the comments where checkContext is declared pure virtual in the Adapter base class. I think it can be improved though, I'll add to it.